addons/web/static/src/views/calendar/calendar_side_panel/calendar_side_panel.scss

```diff
@@ -0,0 +1,7 @@
+.o_calendar_sidebar .o_time_picker input {
+    field-sizing: content;
+}
+
+.o_web_client:has(.o_calendar_sidebar .o_time_picker input.o-dropdown.show) .o-main-components-container .o_time_picker_dropdown {
```

Comment:
mouarf

Reply:
I think you can pass a `class` props to TimePicker, and use it here as a selector

Reply:
Will be done in another PR